tests/GlobalContextTest.php

```diff
@@ -24,6 +24,8 @@ protected function setupTwig(): \Twig\Environment
         Configuration::make($twig)
             ->setTemplatesPath('components')
             ->setTemplatesExtension('twig')
+            ->setComponentsNamespace('\Performing\TwigComponents\Tests\View')
+            ->register('test', \Performing\TwigComponents\Tests\View\Alert::class)
```

Comment:
Would it make sense to always use simple name of the component class as the component name? I don't see much need to have multiple php component classes for a single twig component file 🤔 and it's a bit confusing that you can now use either x-test or x-alert. Imo, this shouldn't be possible.